.gitignore

```diff
@@ -1,3 +1,6 @@
 .idea
 .images
 *.pyc
+test.tif
```

Comment:
use `.git/info/exclude` to exclude local test files and avoid commiting to gitignore